examples/virtual-dom-wip/platform/Html/Internal/Shared.roc

```diff
@@ -0,0 +1,155 @@
+interface Html.Internal.Shared
+    exposes [
+        App,
+        Html,
+        Attribute,
+        CyclicStructureAccessor,
+        Handler,
+        Size,
+        element,
+        text,
+        none,
+        translate,
+        translateStatic,
+        nodeSize,
+    ]
+    imports [
+        Action.{ Action },
+    ]
+
+App state initData : {
+    init : DecodingResult initData -> state,
+    render : state -> Html state,
+    wasmUrl : Str,
+}
+
+DecodingResult a : Result a [Leftover (List U8), TooShort]
+
+Html state : [
+    None,
+    Text Str,
+    Element Str Size (List (Attribute state)) (List (Html state)),
```

Comment:
Can you add a comment as to what `Size` here refers to?

Reply:
Ok sure. It's used on the server side to precalculate the capacity needed for the HTML string. It's commented in the `element` function but yes I will comment on the type too.